akka-http-core/src/test/scala/akka/http/javadsl/HttpExtensionApiSpec.scala

```diff
@@ -13,7 +13,7 @@ import akka.http.javadsl.model.ws._
 import akka.http.javadsl.settings.{ ClientConnectionSettings, ConnectionPoolSettings, ServerSettings }
 import akka.japi.Pair
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/vasseurjb/akka/file/2046322363/issues/source?bid=3132306&fileBranchId=3132371#l14)